components/brave_rewards/resources/page/components/arriving_soon_message.tsx

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+import { MoneyBagIcon } from '../../shared/components/icons/money_bag_icon'
+
+import { formatMessage } from '../../shared/lib/locale_context'
+import { getLocale } from '../../../../common/locale'
+
+import * as styles from './arriving_soon_message.style'
```

Comment:
Nit: Other components use `style` (singular) for these namespace imports, mostly to match the `.style.ts` suffixes (although I've found myself typing "styles" occasionally, so maybe that was a minor mistake).

Reply:
Ahh, fixed.